client/my-sites/media-library/list.jsx

```diff
@@ -1,14 +1,15 @@
 /**
  * External dependencies
  */
-var ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
+var React = require( 'react' ),
 	clone = require( 'lodash/clone' ),
 	noop = require( 'lodash/noop' ),
 	filter = require( 'lodash/filter' ),
 	findIndex = require( 'lodash/findIndex' );
 
+import { AutoSizer, InfiniteLoader, Grid, WindowScroller } from 'react-virtualized';
```

Comment:
Ah I wasn't sure because of @aduth's [comment](https://github.com/Automattic/wp-calypso/pull/10265#discussion_r93803041). :)